packaging/docker/custom/Dockerfile

```diff
@@ -34,7 +34,7 @@ USER grafana
 RUN if [ $GF_INSTALL_IMAGE_RENDERER_PLUGIN = "true" ]; then \
       grafana-cli \
         --pluginsDir "$GF_PATHS_PLUGINS" \
-        --pluginUrl https://github.com/grafana/grafana-image-renderer/releases/latest/download/plugin-linux-x64-glibc-no-chromium.zip \
+        --pluginUrl https://github.com/grafana/grafana-image-renderer/releases/latest/download/plugin-alpine-x64-no-chromium.zip \
```

Comment:
We need to choose the correct download based on the os, we can use the same `if grep -i -q alpine /etc/issue; then \` approach as on line 17 to find out if we're on alpine